### PR TITLE
add support for wo visual backbone

### PR DESCRIPTION
**Pull request by littletomatodonkey**
### PR types
New features

### PR changes
Models

### Description
Support layout model to remove visual backbone for speed-up. Specifially,
* LayoutLMv2(SER 84.5%, 31.46ms) -> LayoutLMv2-wo-backbone(SER 82.1%, 27.80ms)
* LayoutXLM(SER 89.5%, 19.49ms) -> LayoutXLM-wo-backbone(SER 90.5%, 15.49ms)

**Note** The time cost is tested on V100 GPU for trt+fp16

### CODE REVIEW

File: paddlenlp/transformers/layoutlmv2/modeling.py [hunk 1]
```diff
@@ -265,6 +297,8 @@ class LayoutLMv2PretrainedModel(PretrainedModel):
             "https://bj.bcebos.com/paddlenlp/models/transformers/layoutlmv2/layoutlmv2-base-uncased/model_state.pdparams",
             "layoutlmv2-large-uncased":
             "https://bj.bcebos.com/paddlenlp/models/transformers/layoutlmv2/layoutlmv2-large-uncased/model_state.pdparams",
+            "layoutlmv2-wo-backbone-base-uncased":
+            "https://bj.bcebos.com/paddlenlp/models/transformers/layoutlmv2/layoutlmv2-wo-backbone-base-uncased/model_state.pdparams",
```
Comment (yingyibiao) [inline paddlenlp/transformers/layoutlmv2/modeling.py:301]: 这里的权重上传了嘛

**Review by LiuChiachi — APPROVED** [review]
LGTM

File: paddlenlp/transformers/layoutxlm/modeling.py [hunk 2]
```diff
@@ -256,13 +256,48 @@ class LayoutXLMPretrainedModel(PretrainedModel):
...
+            "vocab_size": 250002,
+        },
     }
     resource_files_names = {"model_state": "model_state.pdparams"}
     pretrained_resource_files_map = {
         "model_state": {
             "layoutxlm-base-uncased":
             "https://bj.bcebos.com/paddlenlp/models/transformers/layoutxlm_base/model_state.pdparams",
+            "layoutxlm-wo-backbone-base-uncased":
+            "https://bj.bcebos.com/paddlenlp/models/transformers/layoutxlm_wo_backbone_base/model_state.pdparams",
```
Comment (yingyibiao) [inline paddlenlp/transformers/layoutxlm/modeling.py:300]: 同上

File: paddlenlp/transformers/layoutlmv2/modeling.py (hunk 1 above)
Comment (littletomatodonkey) [reply paddlenlp/transformers/layoutlmv2/modeling.py:301]: 模型均已上传~

**Pull request merged**